tests/dragonfly/snapshot_test.py

```diff
@@ -150,7 +150,14 @@ async def test_dbfilenames(
 
 
 @pytest.mark.asyncio
-@dfly_args({**BASIC_ARGS, "proactor_threads": 4, "dbfilename": "test-redis-load-rdb"})
+@dfly_args(
```

Comment:
why not in rdb_test.cc ?

Reply:
this is a python test that checks that redis can load dragonfly snapshot. what do you want to put in rdb_test.cc?